src/main/js/field/OAuthEndpointButtonField.js

```diff
@@ -0,0 +1,177 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import GeneralButton from 'field/input/GeneralButton';
+import FieldsPopUp from 'field/FieldsPopUp';
+import LabeledField from 'field/LabeledField';
+import * as FieldModelUtilities from 'util/fieldModelUtilities';
+import { createNewConfigurationRequest } from 'util/configurationRequestBuilder';
+import { connect } from 'react-redux';
+import StatusMessage from 'field/StatusMessage';
+import * as HTTErrorUtils from 'util/httpErrorUtilities';
```

Comment:
This is imported with the wrong name

Reply:
The 'as' keyword allows the import of all functions and classes in util/httpErrorUtilities to be references via the name provided after the as keyword.

Reply:
Oh sorry, just meant it was spelled wrong. It should be HTTPErrorUtils